.github/workflows/ci.yaml

```diff
@@ -51,10 +51,10 @@ jobs:
     runs-on: ubuntu-20.04
     steps:
       - name: Checkout source
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
 
       - name: Setup go
-        uses: actions/setup-go@v2
+        uses: actions/setup-go@v3
```

Comment:
setup-go can possibly move to v4, gaining whatever caching improvements are stated [here](https://github.com/marketplace/actions/setup-go-environment)